pkg/conference/track/track.go

```diff
@@ -130,6 +114,15 @@ func NewPublishedTrack[SubscriberID SubscriberIdentifier](
 		defer close(published.done)
 		defer telemetry.End()
 		published.activePublishers.Wait()
+
+		// End any active subscriptions.
+		published.mutex.Lock()
+		published.mutex.Unlock()
```

Comment:
defer?

Reply:
Wow! Thanks! I really missed it, glad that you've found it!

Debugging it once it leads to a race would def took a lot of time 🙂

Fixed.